### ISSUE TITLE
Parcel 2 support

### ISSUE THREAD

**Issue by LasseRosenow** [Cortys/parcel-plugin-webcomponents#2]
Hi, I'm using parcel 2 and it seems like this isn't working with parcel 2.

**Comment by Cortys** [comment]
Yes, Parcel 2 is currently not supported. I plan to look into this once version 2 gets at least a beta release since I'm not sure that no other breaking changes will occur before the final stable release.

**Comment by LasseRosenow** [comment]
@Cortys Parcel 2 is in beta now :)
